eng/helix/InstallNodejs.ps1

```diff
@@ -0,0 +1,52 @@
+[CmdletBinding(PositionalBinding=$false)]
+Param(
+    [ValidateSet("x86","x64","arm64")][string]$Architecture,
+    [string] $Version,
+    [string] $Mirror = "https://nodejs.org/dist",
+    [string] $DestinationFolder
+)
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = "Stop"
+
+if ($IsMacOS) {
```

Comment:
I didn't know about these variables. That seems useful without having to invoke RuntimeInformation.IsOSPlatform